single_iwyu.py

```diff
@@ -35,10 +37,18 @@ def run_cleaned_iwyu (iwyu: List[str], cleaner: List[str], fix_includes: List[st
         _, err = subprocess.Popen(iwyu, stderr=subprocess.PIPE, text=True).communicate()
         out, _ = subprocess.Popen(cleaner, stdin=subprocess.PIPE,
                                   stdout=subprocess.PIPE, text=True).communicate(input=err)
-        subprocess.Popen(fix_includes, stdin=subprocess.PIPE, text=True).communicate(out)
+        if DEBUG:
+            print("DEBUG")
+            print(out)
+        change, _ = subprocess.Popen(fix_includes, stdin=subprocess.PIPE, stdout=subprocess.PIPE, text=True).communicate(out)
     except subprocess.CalledProcessError:
         warn("IWYU FAILED TO RUN")
         return False
+
+    print(change)
```

Comment:
Should you only `print` `change` if `DEBUG` is set?  Otherwise it looks like this changes the output when debug is not enabled?